lib/outbrain/api/relation.rb

```diff
@@ -0,0 +1,16 @@
+module Outbrain
+  module Api
+    class Relation
+      include Enumerable
+      attr_accessor :relations, :totalDataCount, :overAllMetrics, :aggregatedBy 
```

Comment:
going to leave consistent with the API for now. tried aliasing totalDataCount as count and I definitely got it confused with array :count method a couple times.